data/transaction/status_test.go

```diff
@@ -11,112 +11,121 @@ import (
 )
 
 func TestStatusComputer_ComputeStatusWhenInStorageKnowingMiniblock(t *testing.T) {
+	statusComputer := &StatusComputer{
+		SelfShardId: 12,
+	}
+
```

Comment:
why don't you make use of the `NewStatusComputer` you added ?

Reply:
Done!